src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
```

Comment:
Please add access modifiers to your class fields. It's a good practice to make class fields private to follow encapsulation principle of OOP.